### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by hismastersvoice** [spali/go-rscp#27] [General]
Hello, 

I'm new in go, so I struggle a bit with the installation.

I get the issue...
root/go/src/github.com/spali/go-rscp/rscp/constants.go:50:39: syntax error: unexpected b0001000000000000, expecting semicolon or newline or )

Can you give me please advice... 

Thanks in advance.

**Comment by spali** [comment]
Sorry on vacation right now, so can't try to reproduce. But maye line ending problem?
